ext/node/ops/os/cpus.rs

```diff
@@ -246,12 +246,12 @@ pub fn cpu_info() -> Option<Vec<CpuInfo>> {
   let reader = std::io::BufReader::new(fp);
 
   let mut count = 0;
-  for (i, line) in reader.lines().enumerate() {
+  for (i, line) in reader.lines().skip(1).enumerate() {
```

Comment:
if you move the `skip(1)` behind the `enumerate()` then you don't have to add `1` below. Also, can you add a comment explaining why it does this?

Reply:
Oh, actually nevermind. I misread the code.